docs/index.html

```diff
@@ -69,6 +69,8 @@ <h1><a href="http://threejs.org">three.js</a> / docs</h1>
 
 				language = value;
 				createNavigation();
+				updateFilter();
```

Comment:
This line does not seem to have an effect. When changing the language, no filtering is applied and you see all side navigation menu items.

Reply:
when users input something in filter, for example they input geometry. So the menu items shows only about geometry. At that time when they change language, if we don't use updateFilter();, the menu list shows all menu items.